src/Storefront/Resources/app/storefront/src/plugin/cookie/cookie-configuration.plugin.js

```diff
@@ -451,6 +445,36 @@ export default class CookieConfiguration extends Plugin {
         CookieStorage.setItem(cookiePreference, '1', '30');
 
         this._handleUpdateListener(allCookies.map(({ cookie }) => cookie), []);
+    }
+
+    /**
+     * Event handler for the 'Allow all'-button in the cookie bar.
+     *
+     * @private
+     */
+    _acceptAllCookiesFromCookieBar() {
```

Comment:
This method may be ommited in case you add the additional parameter to the `acceptAllCookies`-method as I outlined above.